elementary/monitor/alerts/source_freshness.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+from typing import Optional
+
+from elementary.clients.slack.schema import SlackMessageSchema
+from elementary.monitor.alerts.alert import Alert
+from elementary.utils.log import get_logger
+from elementary.utils.time import try_parse_time
+
+logger = get_logger(__name__)
+
+
+class SourceFreshnessAlert(Alert):
+    TABLE_NAME = "alerts_source_freshness"
+
+    def __init__(
+        self,
+        unique_id: str,
+        snapshotted_at: Optional[str],
+        max_loaded_at: Optional[str],
+        max_loaded_at_time_ago_in_s: Optional[float],
+        source_name: str,
+        identifier: str,
+        freshness_error_after: str,
+        freshness_warn_after: str,
+        freshness_filter: str,
+        path: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.unique_id = unique_id
+        self.snapshotted_at = try_parse_time(snapshotted_at) if snapshotted_at else None
+        self.max_loaded_at = try_parse_time(max_loaded_at) if max_loaded_at else None
+        self.max_loaded_at_time_ago_in_s = max_loaded_at_time_ago_in_s
+        self.source_name = source_name
+        self.identifier = identifier
+        self.freshness_error_after = freshness_error_after
+        self.freshness_warn_after = freshness_warn_after
+        self.freshness_filter = freshness_filter
+        self.path = path
+
+    def to_slack(self, is_slack_workflow: bool = False) -> SlackMessageSchema:
```

Comment:
Alert formatting looks good to me, can you send some examples in the snap channel to hear Talmi's feedback?

Reply:
Already did, sent him in private a couple of days ago.